### PR TITLE
Add setting to show local time

### PR DESCRIPTION
**Pull request by konifar**
It is just for my internationalization session.

If the user is in Singapore, the first session start 11:00 (1 hours later)

2016-02-14_15_33_52_png

The user can set from Setting page.

2016-02-14_15_34_07_png

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java [hunk 1]
```diff
@@ -96,13 +93,4 @@ private void restart() {
...
     }
 
-    private void setNotificationSetting(boolean isChecked) {
-        PrefUtil.put(getContext(), PrefUtil.KEY_NOTIFICATION_SETTING, isChecked);
-    }
-
-    private void switchNotificationSetting() {
-        boolean newValule = !PrefUtil.get(getContext(), PrefUtil.KEY_NOTIFICATION_SETTING, true);
-        setNotificationSetting(newValule);
-        binding.notificationSettingSwitch.setChecked(newValule);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java]: I created new custom view, and move these logic.

File: app/src/main/java/io/github/droidkaigi/confsched/widget/SettingSwitchRowView.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public SettingSwitchRowView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        binding = DataBindingUtil.inflate(LayoutInflater.from(context), R.layout.view_setting_switch_row, this, true);
+
+        if (!isInEditMode()) {
+            TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.SettingSwitchRow);
+
+            try {
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/widget/SettingSwitchRowView.java]: Is this try needed?

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/widget/SettingSwitchRowView.java]: Oh, no need. I'll remove this. Thanks!

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/widget/SettingSwitchRowView.java]: Fixed!

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/widget/SettingSwitchRowView.java]: :100:

**Comment by konifar** [comment]
Can I merge it?

**Comment by hotchemi** [comment]
Ah sorry sure!

**Pull request merged**